docs/general/builtins/scalar-functions.rst

```diff
@@ -2945,6 +2945,32 @@ arguments, the return type is ``numeric``.
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-objects:
+
+Object functions
+================
+
+.. _scalar-object_keys:
+
+``object_keys(object)``
+-----------------------
+
+The ``object_keys`` function returns the set of first level keys of an ``object``.
```

Comment:
Minor: In the issue description we use `skeys` name for the select example and for the `UDF` workaround, in another related issue https://github.com/crate/crate/issues/9358#issuecomment-920863601 as well - but I guess `object_keys` is fine as the argument being processed is neither `hstore` nor `json_object`

Reply:
The behaviour is different from both `json_object_keys` and `s_keys` therefore I would have suggested using `object_keys` as it imho describes best what it does.